src/pages/homepage/homepage.jsx

```diff
@@ -22,7 +23,7 @@ class HomePage extends React.Component {
         return (
             <div className={classes.coloredBackground}>
                 <Header
-                    brand={require("assets/img/himalia-logo.png")}
+                    brand={require("assets/img/adot-logo.png")}
```

Comment:
nit pick but if we're using BackgroundImage above can we just use that variable in here as well?

Reply:
Thank you for pointing this out, this has been fixed.